### DISCUSSION TITLE
Lack of snapping when drawing a line to another existing line.

### DISCUSSION THREAD

**Discussion by Kiliusz** [excalidraw/excalidraw#8466] [General]
Hi, 
Is there an option to turn on snaping to existing lines (on ends) when I'm drawing another line? It seems there is only possibility for snapping when i draw 2 lines and then move one around. This would be huge for me to be able to snap to existing objects while drawing a new line.

**Comment by dwelle** [comment]
not yet, but planned!

**Comment by Kiliusz** [comment]
Thank you for fast answer? any ETA?

**Reply by Mrazator** [reply]
No ETA, but let's track it here https://github.com/excalidraw/excalidraw/issues/8686
